src/Sulu/Bundle/AdminBundle/Resources/js/components/Sticky/sticky.scss

```diff
@@ -0,0 +1,14 @@
+.sticky {
+    position: sticky;
+    top: 0;
+    z-index: 1; /* push element in front of position: relative elements */
```

Comment:
😱

Reply:
this will probably push the sticky element above our overlays/dialogs too 😕  i think we can prevent that from happening by setting `z-index: 0` in the parent component that uses this component, right?